web/src/main/java/com/bazaarvoice/emodb/web/resources/blob/BlobStoreResource1.java

```diff
@@ -139,11 +140,11 @@ public BlobStoreResource1(BlobStore blobStore, Set<String> approvedContentTypes,
                     @Override
                     public Meter load(String key) throws Exception {
                         String metric = new TaggedName.TaggedNameBuilder()
-                                .metricName(metricName)
+                                .metricName(METRIC_PREFIX + metricName)
                                 .addTag("apiKey", key)
                                 .build()
                                 .encode();
-                        return _metricRegistry.meter(MetricRegistry.name("bv.emodb.blob.BlobStoreResource1", metric));
+                        return _metricRegistry.meter(MetricRegistry.name(metric));
```

Comment:
Minor one - No need of MetricRegistry.Name here. You could use it in Line 143, up to you.

Reply:
done